app/controllers/orders_controller.rb

```diff
@@ -21,14 +17,15 @@ def update
     redirect_back(fallback_location: root_path)
   end
 
-
-  def new
-    order = Order.create(status: "ordered", user_id: current_user.id)
+  def create
     item_hash = @cart.cart_items
-    order.add(item_hash)
-    @cart.destroy
+		item_hash.each do |item, quantity|
+      order = Order.create(status: "ordered", user_id: current_user.id, store_id: item.store.id)
```

Comment:
this is probably unnecessary to set the status as ordered, since it's the default value